src/plugins/borrow-plugins/common/BorrowPluginFactory.js

```diff
@@ -0,0 +1,20 @@
+// @flow
```

Comment:
We like to keep a blank line between the `// @flow` comment and the imports. Otherwise the import sorting plugin will think the comment goes with the next import, and move it around.

Reply:
Good to know. I added the space.